wrapper/src/main/java/software/amazon/jdbc/dialect/DialectCodes.java

```diff
@@ -20,9 +20,11 @@ public class DialectCodes {
   public static final String AURORA_MYSQL = "aurora-mysql";
   public static final String RDS_MYSQL = "rds-mysql";
   public static final String MYSQL = "mysql";
+  public static final String RDS_MULTI_AZ_MYSQL_CLUSTER = "rds-multi-az-mysql-cluster";
```

Comment:
Can we be a more consistent with previous names and remove "_cluster"?

Reply:
`Cluster` is part of the product name and we can't remove it.
[MultiAZ DB Cluster](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/multi-az-db-clusters-concepts.html) consists of 3 nodes, while [MultiAZ DB](https://aws.amazon.com/rds/features/multi-az/) consists of 1 active node and 1 standby node.